google-service-api-json-key.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
That was my fault, I knew this but this one escaped my attention. Fortunately this key works only for my personal account and the only authorization I gave is read access to a mocked spreadsheet. So no harm done.